packages/tldraw/src/lib/ui/components/ActionsMenu/DefaultActionsMenu.tsx

```diff
@@ -47,27 +48,30 @@ export const DefaultActionsMenu = memo(function DefaultActionsMenu({
 	return (
 		<TldrawUiPopover id="actions-menu">
 			<TldrawUiPopoverTrigger>
-				<TldrawUiButton
-					type="icon"
-					data-testid="actions-menu.button"
-					title={msg('actions-menu.title')}
-				>
-					<TldrawUiButtonIcon icon="dots-vertical" small />
-				</TldrawUiButton>
+				<PopoverPrimitive.Anchor>
```

Comment:
I thought `Anchor` was only needed when the anchor was something different to the trigger? Does `TldrawUiPopoverTrigger` not imply this?

Reply:
ah sorry, this was leftover from some testing i was doing, removing!